From: Dermot Bradley <dermot_bradley@yahoo.com>
Date: Thu, 14 Jan 2021 17:41 +0000
Subject: [PATCH] Correction for skalibs ABI change

The recent skalibs version update included an abi change and this
package was not rebuilt at the time so the issue was not spotted.

---

diff -aur a/src/justc-envdir/justc-envdir.c b/src/justc-envdir/justc-envdir.c
--- a/src/justc-envdir/justc-envdir.c
+++ b/src/justc-envdir/justc-envdir.c
@@ -9,7 +9,7 @@
 #include <skalibs/env.h>
 #include <skalibs/direntry.h>
 #include <skalibs/stralloc.h>
-#include <skalibs/djbunix.h>
+#include <skalibs/stddjb.h>
 
 #define USAGE "justc-envdir [-I | -i ] dir prog..."
 
@@ -36,7 +36,7 @@
   if (argc < 2) strerr_dieusage(100, USAGE) ;
   if ((justc_envdir(*argv++, &modifs) < 0) && (insist || (errno != ENOENT)))
     strerr_diefu1sys(111, "justc_envdir") ;
-  xpathexec_r(argv, envp, env_len(envp), modifs.s, modifs.len) ;
+  xmexec_fm(argv, envp, env_len(envp), modifs.s, modifs.len) ;
 }
 
 static int justc_envdir (char const *path, stralloc *modifs)
